python/my-widget-service/README.md

```diff
@@ -0,0 +1,58 @@
+
+# Welcome to your CDK Python project!
+
+This is a blank project for Python development with CDK.
```

Comment:
This file isn't normally included with the example apps on this repo. If it is included, it needs updating to not still have the default text, e.g. 'This is a blank project...'

Reply:
I'll add the info. I would be hesitant to use the fact that READMEs haven't been used in the past to justify not including them just to be consistent. I'm personally biased towards a readme for each directory.

Reply:
The main reason that the existing examples don't have a README is that they were all generated prior to the addition of Python-specific init templates in the CDK CLI tool.  I think the addition of README files is good, although as mentioned, they should be edited to make them relevant to the example.